app/javascript/dashboard/routes/dashboard/settings/automation/constants.js

```diff
@@ -283,53 +214,102 @@ export const AUTOMATIONS = {
+      { key: 'send_attachment', name: 'SEND_ATTACHMENT' },
+    ],
+  },
+  conversation_opened: {
+    conditions: [
       {
-        key: 'assign_agent',
-        name: 'ASSIGN_AGENT',
+        key: 'browser_language',
+        name: 'BROWSER_LANGUAGE',
+        inputType: 'search_select',
+        filterOperators: OPERATOR_TYPES_1,
       },
       {
-        key: 'assign_team',
-        name: 'ASSIGN_TEAM',
+        key: 'email',
+        name: 'EMAIL',
+        inputType: 'plain_text',
+        filterOperators: OPERATOR_TYPES_2,
       },
       {
-        key: 'assign_agent',
-        name: 'ASSIGN_AGENT',
+        key: 'mail_subject',
+        name: 'MAIL_SUBJECT',
+        inputType: 'plain_text',
+        filterOperators: OPERATOR_TYPES_2,
       },
       {
-        key: 'send_email_to_team',
-        name: 'SEND_EMAIL_TO_TEAM',
+        key: 'country_code',
+        name: 'COUNTRY_NAME',
+        inputType: 'search_select',
+        filterOperators: OPERATOR_TYPES_1,
       },
       {
-        key: 'send_message',
-        name: 'SEND_MESSAGE',
+        key: 'referer',
+        name: 'REFERER_LINK',
+        inputType: 'plain_text',
+        filterOperators: OPERATOR_TYPES_2,
       },
       {
-        key: 'send_email_transcript',
-        name: 'SEND_EMAIL_TRANSCRIPT',
+        key: 'assignee_id',
+        name: 'ASSIGNEE_NAME',
+        inputType: 'search_select',
+        filterOperators: OPERATOR_TYPES_3,
       },
       {
-        key: 'mute_conversation',
-        name: 'MUTE_CONVERSATION',
+        key: 'phone_number',
+        name: 'PHONE_NUMBER',
+        inputType: 'plain_text',
+        filterOperators: OPERATOR_TYPES_6,
       },
       {
-        key: 'snooze_conversation',
-        name: 'SNOOZE_CONVERSATION',
+        key: 'team_id',
+        name: 'TEAM_NAME',
+        inputType: 'search_select',
+        filterOperators: OPERATOR_TYPES_3,
       },
       {
-        key: 'resolve_conversation',
-        name: 'RESOLVE_CONVERSATION',
+        key: 'inbox_id',
+        name: 'INBOX',
+        inputType: 'multi_select',
+        filterOperators: OPERATOR_TYPES_1,
       },
       {
-        key: 'send_webhook_event',
-        name: 'SEND_WEBHOOK_EVENT',
+        key: 'conversation_language',
+        name: 'CONVERSATION_LANGUAGE',
+        inputType: 'multi_select',
+        filterOperators: OPERATOR_TYPES_1,
       },
       {
-        key: 'send_attachment',
-        name: 'SEND_ATTACHMENT',
+        key: 'priority',
+        name: 'PRIORITY',
+        inputType: 'multi_select',
+        filterOperators: OPERATOR_TYPES_1,
       },
     ],
+    actions: [
+      { key: 'assign_agent', name: 'ASSIGN_AGENT' },
+      { key: 'assign_team', name: 'ASSIGN_TEAM' },
+      { key: 'assign_agent', name: 'ASSIGN_AGENT' },
+      { key: 'send_email_to_team', name: 'SEND_EMAIL_TO_TEAM' },
+      { key: 'send_message', name: 'SEND_MESSAGE' },
+      { key: 'send_email_transcript', name: 'SEND_EMAIL_TRANSCRIPT' },
```

Comment:
The action list includes actions that don't make sense after resolution:
  - mute_conversation - pointless on resolved conversations
  - snooze_conversation - can't snooze resolved conversations
  - assign_agent - typically unnecessary post-resolution

Reply:
@sojan-official @scmmishra Thoughts?